Assets/Plugins/iOS/Editor/IOSBuilder.cs

```diff
@@ -46,14 +64,27 @@ public override void PostBuild(BuildReport report)
             string targetGUID = proj.GetUnityMainTargetGuid();
             string unityTargetGUID = proj.GetUnityFrameworkTargetGuid();
 
-            proj.SetBuildProperty(targetGUID, "ENABLE_BITCODE", "false");
-            proj.SetBuildProperty(unityTargetGUID, "ENABLE_BITCODE", "false");
+            proj.SetBuildProperty(
```

Comment:
I'm not certain if I like this style change.

Reply:
No problem, I reverted to the previous style of having it all on one line. I only realized in retrospect that I had not made separate commits between the style change and the actual fix. Ideally, they would be two different tasks - but decoupling them seemed like more work than it was worth. Easy to fix small preferences like this though.